appveyor.yml

```diff
@@ -0,0 +1,60 @@
+
+os: Visual Studio 2015
+
+environment:
+  matrix:
+    - ARCH:                x64
+      D_COMPILER:          dmd
+      D_VERSION:           2.077.1
+      BRANCH:              master
+
+cache:
+
+init:
+  - git config --global core.autocrlf input
+
+build_script:
+  - cd c:/projects/
+  - ps: |
+        Start-FileDownload "http://downloads.dlang.org/releases/2.x/$Env:D_VERSION/dmd.$Env:D_VERSION.windows.7z" -FileName dmd2.7z
+        7z x dmd2.7z > $null
+        Start-FileDownload "http://downloads.dlang.org/other/dm857c.zip" -FileName dm857c.zip
+        7z x dm857c.zip > $null
+        Start-FileDownload "http://ftp.digitalmars.com/bup.zip" -FileName bup.zip
+        7z x bup.zip dm\bin\implib.exe > $null
+        Start-FileDownload "http://semitwist.com/download/app/dmd-localextras.7z" -FileName extras.7z
```

Comment:
That's hasn't been used in ages, you can take the binaries from the old release.

Reply:
> That's hasn't been used in ages, you can take the binaries from the old release.

How do you get anything new into that, e.g. link.exe or snn.lib?